x/claim/spec/README.md

```diff
@@ -0,0 +1,42 @@
+# `claim`
+
+## Abstract
+
+This specifies the `claim` module of Osmosis project, provide commands for claimable amount query and claim airdrop.
+We apply real-time decay after `DurationUntilDecay` pass where monthly decay rate is `-10%` of inital airdrop amount.
+When `DurationOfDecay` time pass, all coins are funded to community pool.
+
+## Genesis State
+
+### Accounts
+
+All genesis accounts have `1 Osmo` for claim fee.
+
+### Claimables
+
+Claimables are configured by genesis.
```

Comment:
Can you define what a claimable is here?

Reply:
ok